baseapp-auth/baseapp_auth/rest_framework/register/serializers.py

```diff
@@ -9,8 +9,7 @@
 
 
 class RegisterSerializer(serializers.Serializer):
-    first_name = serializers.CharField(required=False, allow_blank=True)
-    last_name = serializers.CharField(required=False, allow_blank=True)
+    name = serializers.CharField(required=False, allow_blank=True)
```

Comment:
any reasons why we are changing it like that? Not sure about the advantages of it...

Reply:
@vitorguima we are changing the frontend to only ask for "name" instead of both first and last, I don't want to change the user model now so it is a step before actually changing the user model as well